docs/requirement-sets/outlook/outlook-api-requirement-sets.md

```diff
@@ -8,23 +8,9 @@ ms.localizationpriority: high
 
 # Outlook JavaScript API requirement sets
 
-Outlook add-ins declare what API versions they require in their manifest. The markup varies depending on whether you're using the [XML manifest format](/office/dev/add-ins/develop/xml-manifest-overview) or the [unified manifest for Microsoft 365 (preview)](/office/dev/add-ins/develop/unified-manifest-overview).
+Outlook add-ins declare what API versions they require in their manifest. The markup varies depending on whether you're using the [XML manifest format](/office/dev/add-ins/develop/xml-manifest-overview) or the [unified manifest for Microsoft 365](/office/dev/add-ins/develop/unified-manifest-overview).
```

Comment:
Consider changing the order here to match the tab and preference order.
```suggestion
Outlook add-ins declare what API versions they require in their manifest. The markup varies depending on whether you're using the [unified manifest for Microsoft 365](/office/dev/add-ins/develop/unified-manifest-overview) or the [XML manifest format](/office/dev/add-ins/develop/xml-manifest-overview).
```

Reply:
I already merged. But I'll make a note to fix this soon.